docs/install.md

```diff
@@ -11,6 +11,8 @@ Ubuntu installation [here](https://docs.docker.com/v17.12/install/linux/docker-c
 If you have a CUDA GPU, you can use [Nvidia Docker](https://github.com/NVIDIA/nvidia-docker)
 instead of Docker to make certain analysers more performant.
 
+NOTE (05/2023): if you are on Apple Silicon, your machine will not be able to natively run FA's docker images. In order to fix this, you'll need to [enable virtualization](https://collabnix.com/warning-the-requested-images-platform-linux-amd64-does-not-match-the-detected-host-platform-linux-arm64-v8/) by changing some settings in Docker Desktop. Navigate to Settings > General and make sure the "Use Virtualization framework" box is checked. After, navigate to Settings > Features in development and check the "Use Rosetta for x86/amd64 emulation on Apple Silicon." If you're well-versed in Docker, you can set the 'platform' flag to 'linux/amd64' in the Dockerfile. If not, the easiest solution is to modify your personal ~/.bashrc or ~/.zshrc file and add ``export DOCKER_DEFAULT_PLATFORM=linux/amd64`` to it. 
```

Comment:
Thanks, this is a good comment. We could also look into building a Docker image that is compatible with Apple Silicon.